### DISCUSSION TITLE
Losing @unique(cuid)?

### DISCUSSION THREAD

**Discussion by darrylmorley** [prisma/prisma#9385] [Q&A] [answered]
When I use `prisma db pull` to start a new project with my current db, the new schema.prisma omits the `@unique(cuid())` from one of my id fields. 

model Log in new app, generated with `prisma db pull`:
```prisma
model Log {
  id    String @id
  time  String
  error String
  site  String
  level String
}
```

model Log from initial setup:
```prisma
model Log {
  id    String @id @default(cuid())
  time  String
  error String
  site  String
  level String
}
```

If I try to re-add the `@default(cuid())` then prisma wants to wipe my database and start again?

Am I doing something wrong here? I'm new to using Prisma.

**Comment by ryands17** [comment, marked as answer]
@darrylmorley 👋 

This seems like a bug. It would be great if you could create an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that we can look into this 🙂

**Reply by darrylmorley** [reply]
Thanks @ryands17, will do.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
